shlr/gdb/src/packet.c

```diff
@@ -159,7 +159,14 @@ int read_packet(libgdbr_t *g) {
 		}
 	}
 	g->data_len = 0;
-	while (r_socket_ready (g->sock, 0, READ_TIMEOUT) > 0) {
+	while (1) {
```

Comment:
what about adding a counter and abort after N retries? infinite loops are scary

Reply:
Okay, but the counter won't work for vcont packets. Because the target might be waiting for IO.